engines/dialogs.cpp

```diff
@@ -78,9 +102,10 @@ MainMenuDialog::MainMenuDialog(Engine *engine)
 	GUI::StaticTextWidget *title = new GUI::StaticTextWidget(this, "GlobalMenu.Title", "ScummVM");
 	title->setAlign(Graphics::kTextAlignCenter);
 #endif
-
-	GUI::StaticTextWidget *version = new GUI::StaticTextWidget(this, "GlobalMenu.Version", gScummVMVersionDate);
+     
+	GUI::StaticTextWidget *version = new GUI::StaticTextWidget(this, "GlobalMenu.Version", caption.c_str()) ;
```

Comment:
I removed the version name in the GMM and instead of that i added the game name. The reason behind doing so was that the version name is already displayed in the beginning when we launch ScummVM.exe , hence i thought that it would be useful if the game title is displayed when the user pauses the game by pressing Ctrl+F5. Displaying the game title would be useful in identifying the game which has been paused.